windows-packaging/CalicoWindows/kubernetes/kube-proxy-service.ps1

```diff
@@ -16,10 +16,12 @@ Param(
     [string]$NetworkName = "Calico"
 )
 
+$CalicoPath=(Resolve-Path ..).Path

```

Comment:
I wonder if we need to ensure that this script will be called from the directory we expect.

Maybe we need to check if https://github.com/projectcalico/node/blob/master/windows-packaging/CalicoWindows/kubernetes/install-kube-services.ps1 is run from the right directory (`c:\CalicoWindows/kubernetes/`)

Reply:
IIUC Resolve-Path gets the value `..` relative to the current directory that the script is invoked from

Reply:
We are using relative directory here because the assumption is `kubernetes` directory located at the same level with `config.ps1` and other Calico files. All the files been unzipped from a single archive so it should always be running from the right directory.